lib/App/Config/Chronicle.pm

```diff
@@ -866,4 +870,15 @@ L<http://search.cpan.org/dist/App-Config/>
 
 =cut
 
+# helper on "warnings" scope
+## no critic (ProhibitMultiplePackages, ProhibitNoWarnings)
+package helper {
```

Comment:
Can't do this, unfortunately: it'll register a new package on CPAN called `helper`.

If the deprecation warning is causing trouble, I would suggest split this out into an internal method, and a wrapper public method which does the deprecation warning then calls the internal method:

```
sub save_dynamic {
 my $self = shift;
 warnings::warnif deprecated => ...;
 $self->_save_dynamic(@_)
}
```

and have all the code in this module call `_save_dynamic` instead.

Reply:
Indeed, completely forgot this one will be in CPAN.